lib/galaxy/datatypes/converters/vcf_bgzip_to_vcf_converter.xml

```diff
@@ -0,0 +1,12 @@
+<tool id="CONVERTER_vcf_bgzip_to_vcf_0" name="Convert VCF_BGZIP to VCF" version="1.0.0" hidden="true">
+    <!-- <description>__NOT_USED_CURRENTLY_FOR_CONVERTERS__</description> -->
+    <command>python '$__tool_directory__/vcf_bgzip_to_vcf.py' '$input1' '$output1'</command>
+    <inputs>
+        <param format="vcf_bgzip" name="input1" type="data" label="Choose VCF BGZip file"/>
+    </inputs>
+    <outputs>
+        <data format="vcf" name="output1"/>
+    </outputs>
```

Comment:
Can you add a test here ? You should be able to use `test.vcf.gz` as input

Reply:
I unpacked the `test.vcf.gz` file in `test-data` directory and added it as `test.vcf`, then added the test as
```
      <test>
        <param name="input1" ftype="vcf_bgzip" value="test.vcf.gz"/>
        <output name="output1" file="test.vcf"/>
      </test>
```
 following https://galaxyproject.org/admin/tools/writing-tests/
(This is the first time I write a test for a tool, so lmk if I need to add/do something else)

Reply:
Yep, that should work. To use the converter you also need to add a line like
```
<converter file="vcf_bgzip_to_vcf_converter.xml" target_datatype="vcf"/>
```
to the `vcf_bgzip ` datatype in config/datatypes_conf.xml.sample.

Reply:
Done in https://github.com/galaxyproject/galaxy/pull/7926/commits/4d353df7813382bc3374b6217f82f578b1618694

Reply:
The test is failing with error:
```
JobOutputsError: History item  different than expected, difference (using diff): ( /tmp/tmp_VSCQGtest.vcf v. /tmp/tmpqTVMR2test.vcf ) --- local_file +++ history_data @@ -1,34 +1,34 @@ -##fileformat=VCFv4.1 -##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype"> -##INFO=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth; some reads may have been filtered"> -#CHROM POS ID REF ALT QUAL FILTER INFO FORMAT wgs2 -chr1 500 . C T . . DP=10 GT 1/1 -chr1 1000 . A G . . DP=10 GT 0/1 -chr1 1500 . G A . . DP=10 GT 1/0 -chr1 2000 . G A . . DP=10 GT 1/1 -chr1 2500 . A G . . DP=10 GT 0/1 -chr1 3000 . G C . . DP=10 GT 1/0 -chr1 3500 . G T . . DP=10 GT 1/1 -chr1 4000 . A G . . DP=10 GT 0/1 -chr1 4500 . A C . . DP=10 GT 1/0 -chr1 5000 . A G . . DP=10 GT 1/1 -chr1 5500 . C T . . DP=10 GT 0/1 -chr1 6000 . C A . . DP=10 GT 1/0 -chr1 6500 . C T . . DP=10 GT 1/1 -chr1 7000 . G T . . DP=10 GT 0/1 -chr1 7500 . G C . . DP=10 GT 1/0 -chr1 8000 . T G . . DP=10 GT 1/1 -chr1 8500 . T A . . DP=10 GT 0/1 -chr1 9000 . C T . . DP=10 GT 1/0 ******** *SNIP * ******** +b'chr1\t3000\t.\tG\tC\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t3500\t.\tG\tT\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t4000\t.\tA\tG\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t4500\t.\tA\tC\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t5000\t.\tA\tG\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t5500\t.\tC\tT\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t6000\t.\tC\tA\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t6500\t.\tC\tT\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t7000\t.\tG\tT\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t7500\t.\tG\tC\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t8000\t.\tT\tG\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t8500\t.\tT\tA\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t9000\t.\tC\tT\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t9500\t.\tG\tA\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t10000\t.\tA\tG\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t10500\t.\tT\tC\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t11000\t.\tA\tG\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t11500\t.\tC\tT\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t12000\t.\tA\tG\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t12500\t.\tC\tG\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t13000\t.\tT\tC\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t13500\t.\tG\tT\t.\t.\tDP=10\tGT\t1/0' +b'chr1\t14000\t.\tA\tC\t.\t.\tDP=10\tGT\t1/1' +b'chr1\t14500\t.\tA\tG\t.\t.\tDP=10\tGT\t0/1' +b'chr1\t15000\t.\tT\tG\t.\t.\tDP=10\tGT\t1/0'
```
I tried to manually run the script and it output the same file that i added under `test-data/test.vcf` (diff returns without output), so not sure why this is happening (i'm assuming I did something wrong with the test portion of the xml but not sure what yet). Will look more into it but if the problem is obvious to anyone, input would be greatly appreciated!

Reply:
Just as a tip, you can run those framework tests like this:
```
sh run_tests.sh -framework -id CONVERTER_vcf_bgzip_to_vcf
```
Weirdly this is passing locally: 
```
collected 220 items / 219 deselected / 1 selected

test/functional/test_toolbox_pytest.py::test_tool[CONVERTER_vcf_bgzip_to_vcf_0_test_1] PASSED [100%]
```